### PR TITLE
doc: initial ChatQnA description and deployment

### PR DESCRIPTION
**Pull request by dbkinder**
Fixed issues with Tom's original PR #84 so we can continue working. We need to merge this WIP so others can contribute.

### CODE REVIEW

File: examples/ChatQnA/deploy/AIPC.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.. _ChatQnA_deploy_aiPC:
+
+
+Single Node On-Prem Deployment: AI PC
+#####################################
```
Comment (xiguiw) [inline examples/ChatQnA/deploy/AIPC.rst:5]: Do we use markdown or rst format?

Comment (dbkinder) [reply examples/ChatQnA/deploy/AIPC.rst:5]: In general you can use either reST or markdown for content.

File: examples/ChatQnA/ChatQnA_Guide.rst [hunk 2]
```diff
@@ -6,86 +6,110 @@ ChatQnA Sample Guide
...
+#. **LLMs**: The retrieved data points are then passed to large language models (LLMs) for further processing. LLMs are powerful generative models that have been trained on a large corpus of text data. They can generate human-like responses based on the input data.
+#. **Generate Response**: The LLMs generate a response based on the input data and the user query. This response is then returned to the user as the chatbot's answer.
 
 Expected Output
 ===============
 
 Validation Matrix and Prerequisites
 ***********************************
 
+See :doc:`/GenAIExamples/supported_examples`
```
[the remark this answers is not in the document]
Comment (dbkinder) [reply examples/ChatQnA/ChatQnA_Guide.rst:67]: This is how you create a link from a reST to a Markdown doc (you don't include the extension).

File: examples/index.rst [hunk 3]
```diff
@@ -8,6 +8,13 @@ testing, and scalability. All examples are fully compatible with Docker and
 Kubernetes, supporting a wide range of hardware platforms such as Gaudi, Xeon,
 and other hardwares.
 
+.. toctree::
+   :maxdepth: 1
+
+   ChatQnA/ChatQnA_Guide
```
[the remark this answers is not in the document]
Comment (dbkinder) [reply examples/index.rst:14]: In a toc-tree directive you link to the file without the extension.

**Review by kevinintel — APPROVED** [review]

**Comment by dbkinder** [comment]
Closing this to open another

**Pull request closed without merging**